util/MySqlMigrations/Scripts/2022-03-01_00_Down_MigrateOrganizationApiKeys.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE Organization
```

Comment:
This pattern doesn't work with existing scripts we have. We would run the down then up script, which I guess might work, but doesn't really jive with what we have.

Reply:
I'm confused with what those scripts are for. Two of them are manual migration steps that don't happen within a transaction and don't update the migration table and the rest are the full migration in `sql`/`psql` form. If someone were to only run the migration via that folder their database wouldn't know that they applied all the migrations. 

For us developers we should be applying the migrations the the `dotnet ef` right? Which wouldn't use those files at all and do we have instructions for how end users should apply migrations for EF databases? Shouldn't they just run an idempotent script that we generate using `dotnet ef migrations script --idempotent` and check into source?

Reply:
We don't have a standard way to migrate EF data, but if it were to user `dotnet ef` I don't see what these migration scripts are good for.

At the moment, it's intentionally designed to emulate how we handle dapper migrations. We should continue to do that until we know for sure we will be migrating EF through some other mechanism.